ext/system-resources/resources/chat/cl_chat.lua

```diff
@@ -227,34 +227,30 @@ local CHAT_HIDE_STATES = {
 local kvpEntry = GetResourceKvpString('hideState')
 local chatHideState = kvpEntry and tonumber(kvpEntry) or CHAT_HIDE_STATES.SHOW_WHEN_ACTIVE
 local isFirstHide = true
-
-if not isRDR then
```

Comment:
this check is redundante  since we are already checking for if RegisterKeyMapping exists, so best to remove it.